### ISSUE TITLE
Serialport provider error handling broken?

### ISSUE THREAD

**Issue by hkapanen** [SignalK/signalk-server-node#96]
Following log excerpt illustrates the problem:

```
Jun 11 19:20:32 sophia signalk-server[7438]: signalk-server running at 0.0.0.0:3000
Jun 11 19:20:32 sophia signalk-server[7438]: /opt/signalk-server/providers/serialport.js:38
Jun 11 19:20:32 sophia signalk-server[7438]: this.serial.unpipe(this);
Jun 11 19:20:32 sophia signalk-server[7438]: ^
Jun 11 19:20:32 sophia signalk-server[7438]: TypeError: this.serial.unpipe is not a function
Jun 11 19:20:32 sophia signalk-server[7438]: at SerialStream.start (/opt/signalk-server/providers/serialport.js:38:17)
Jun 11 19:20:32 sophia signalk-server[7438]: at emitOne (events.js:77:13)
Jun 11 19:20:32 sophia signalk-server[7438]: at SerialPort.emit (events.js:169:7)
Jun 11 19:20:32 sophia signalk-server[7438]: at SerialPort.<anonymous> (/opt/signalk-server/node_modules/serialport/lib/serialport.js:103:14)
Jun 11 19:20:32 sophia signalk-server[7438]: at SerialPort._error (/opt/signalk-server/node_modules/serialport/lib/serialport.js:186:5)
Jun 11 19:20:32 sophia signalk-server[7438]: at /opt/signalk-server/node_modules/serialport/lib/serialport.js:210:19
Jun 11 19:20:32 sophia systemd[1]: signalk-server.service: main process exited, code=exited, status=1/FAILURE
```

**Comment by hkapanen** [comment]
After a suggestion by @tkurki, changing...
`https://github.com/SignalK/signalk-server-node/blob/master/providers/serialport.js#L56` 
...into...
`this.serial.on('error', function(x) {console.log(x)});`
...exposes the error:
```
Jun 12 20:35:48 sophia signalk-server[17149]: signalk-server running at 0.0.0.0:[object Object]
Jun 12 20:35:48 sophia signalk-server[17149]: [Error: Cannot open  /dev/gps]
Jun 12 20:35:48 sophia signalk-server[17149]: [Error: Cannot open  /dev/nexus]
Jun 12 20:35:59 sophia systemd[1]: Reloading.
```

**Comment by tkurki** [comment]
fixed by 7da8e61b
